.github/workflows/setup-kicad-library-utils.yml

```diff
@@ -0,0 +1,52 @@
+name: setup-kicad-library-utils
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test-default:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v5.0.0
+
+      - name: Run action (default version)
```

Comment:
- Stick to `Run action` as the step name for both jobs, no need for extra context
- Job name wise, stick to `test` and `test-version`